lib/utils/error-message.js

```diff
@@ -154,10 +154,12 @@ function errorMessage (er) {
       var msg = er.message.replace(/^404\s+/, '')
       short.push(['404', msg])
       if (er.pkgid && er.pkgid !== '-') {
+        var pkg = er.pkgid.replace(/(?!^)@.*$/, '')
+
         detail.push(['404', ''])
-        detail.push(['404', '', "'" + er.pkgid + "' is not in the npm registry."])
+        detail.push(['404', '', "'" + pkg + "' is not in the npm registry."])
```

Comment:
Output should still include the version, because if I say `npm i npm@20.0.0` I don't want an error saying that `npm` isn't in the registry.

Reply:
Seems I forgot to address this earlier, but I added a fix (not sure why I changed it in the first place).

Reply:
Seeing this makes me want to use template literals. =)